packages/web-runtime/src/index.ts

```diff
@@ -82,6 +83,9 @@ export const renderSuccess = (): void => {
     wormhole: instance.config.globalProperties.$wormhole
   })
 
+  // @vueuse/head
```

Comment:
What's this comment needed for?

Reply:
Clarified in a call: just for consistency, because the other blocks also have a comment.